.github/workflows/linters.yml

```diff
@@ -6,32 +6,20 @@ env:
   FORCE_COLOR: 1
 
 jobs:
-  lighthouse:
-    name: Lighthouse
+  eslint:
+    name: ESLint
     runs-on: ubuntu-18.04
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v1
         with:
           node-version: "12.x"
-      - name: Setup Lighthouse
-        run: npm install -g @lhci/cli@0.7.x
-      - name: Lighthouse Report
-        run: lhci autorun --upload.target=temporary-public-storage --collect.staticDistDir=.
-  webhint:
-    name: Webhint
```

Comment:
- [ ] Great job so far, one of the project requirements is to add [JavaScript + HTML + CSS linter](https://github.com/microverseinc/linters-config/tree/master/html-css-js)., kindly fix that.

Reply:
Thankyou @Wes-Isaac  Its fixed now! 👍